src/content/1.7/modules/creation/displaying_content_in_front_office.md

```diff
@@ -260,26 +268,33 @@ which is a must when using the translation tool.
 
 Here are our two files:
 
-    display.php
-    <?php
-    class mymoduledisplayModuleFrontController extends ModuleFrontController
+- *display.php*
+
+```php
+class mymoduledisplayModuleFrontController extends ModuleFrontController
+{
+    public function initContent()
     {
-      public function initContent()
-      {
         parent::initContent();
         $this->setTemplate('module:mymodule/views/templates/front/display.tpl');
-      }
     }
+}
+```
 
-    display.tpl
-    Welcome to my shop!
+- **display.tpl*
+```
+Welcome to my shop!
+```
```

Comment:
One carriage return is missing, we get unexpected results

![capture du 2018-04-26 17-42-08](https://user-images.githubusercontent.com/6768917/39319526-3a01deac-4979-11e8-998c-831c3c0cc1a5.png)

Reply:
Look weird, didn't have this result on my hugo instance :/